pulumi/folio/index.ts

```diff
@@ -268,11 +289,15 @@ const modules: FolioModule[] = folio.prepare.moduleList(folioDeployment);
 
 // Get a reference to the okapi module.
 const okapi: FolioModule = util.getModuleByName("okapi", modules);
+
+// TODO Use the constructor for folio module or add other required props here.
+// Or maybe just create create its own function to deploy it since it is sort of its
+// own animal. Its also not a module so a lot of things are different about it.
 // Create a reference to the stripes module,
-const stripes: FolioModule = {
-    name: "stripes",
-    version: "2021.r2.2"
-}
+// const stripes: FolioModule = {
+//     name: "stripes",
+//     version: "2021.r2.2"
+// }
```

Comment:
Do you want to create a follow up ticket to put stripes back in later?

Reply:
Done: https://github.com/culibraries/folio/issues/142. Also created issue for the LB stuff.